pkg/volume/util/atomic_writer.go

```diff
@@ -341,6 +341,12 @@ func (w *AtomicWriter) newTimestampDir() (string, error) {
 		return "", err
 	}
 
+	err = os.Chmod(tsDir, 0655)
```

Comment:
We could get away with 0655 since owner is always going to be root and will have dac override capability, but 0755 is more correct, probably.